### PR TITLE
PBS Index Atag changes

### PR DESCRIPTION
**Pull request by spotxslagle**
## Type of change

- [ x ] Feature

## Description of change

Atag changes for PBS Adapter

### CODE REVIEW

File: modules/prebidServerBidAdapter/index.js [hunk 1]
```diff
@@ -475,7 +475,9 @@ export function PrebidServer() {
           if (isValid) {
             bidRequests.forEach(bidderRequest => events.emit(EVENTS.BIDDER_DONE, bidderRequest));
           }
-          if (shouldEmitNonbids(s2sBidRequest.s2sConfig, response)) {
+          const { emitSeatNonBid, emitAtag } = shouldEmitAnalyticsEvent(s2sBidRequest.s2sConfig, response)
+          // should be deprecated in Prebid 10
```
Comment (patmmccann) [inline modules/prebidServerBidAdapter/index.js:479]: can you elaborate?

Comment (spotxslagle) [reply modules/prebidServerBidAdapter/index.js:479]: We would like to deprecate the seat_non_bid event in favor of the pbs_analytics event. That way adapters only need to watch for the one event.

**Comment by patmmccann** [comment]
fixes https://github.com/prebid/Prebid.js/issues/11391

**Comment by spotxslagle** [comment]
Superceded by https://github.com/prebid/Prebid.js/pull/12044

**Pull request closed without merging**